From: Niklas Cassel <niklas.cassel@axis.com>
Date: Thu, 16 Nov 2017 18:38:06 +0100
Subject: [PATCH] scsi: pmcraid: use correct size unit when calling
 find_first_zero_bit()
References: bsc#1077989
Git-commit: 36d9e0e8a77cfe6983940a266288f7c9d34c0b37
Patch-mainline: v4.16-rc1

find_first_zero_bit()'s parameter 'size' is defined in bits, not in
bytes.

Signed-off-by: Niklas Cassel <niklas.cassel@axis.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/pmcraid.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/scsi/pmcraid.c b/drivers/scsi/pmcraid.c
index 11d1a2d3d58a..512afe207143 100644
--- a/drivers/scsi/pmcraid.c
+++ b/drivers/scsi/pmcraid.c
@@ -5223,7 +5223,7 @@ static unsigned short pmcraid_get_minor(void)
 {
 	int minor;
 
-	minor = find_first_zero_bit(pmcraid_minor, sizeof(pmcraid_minor));
+	minor = find_first_zero_bit(pmcraid_minor, PMCRAID_MAX_ADAPTERS);
 	__set_bit(minor, pmcraid_minor);
 	return minor;
 }
-- 
2.12.3

